lib/bokken/accounts/user.ex

```diff
@@ -10,13 +10,15 @@ defmodule Bokken.Accounts.User do
   @roles [:ninja, :guardian, :mentor, :organizer]
 
   @required_fields [:email, :password, :role]
-  @optional_fields [:active, :verified, :registered]
+  @optional_fields [:active, :verified, :registered, :discord_id]
 
   schema "users" do
     field :email, :string
     field :password_hash, :string
     field :password, :string, virtual: true
 
+    field :discord_id, :string
```

Comment:
It would be nice, in the future, to tie this to the user socials (specifically discord one). What's your opinion? 🙏🏼